pkg/sqlite/load.go

```diff
@@ -1320,9 +1320,10 @@ func (s *sqlLoader) rmChannelEntry(tx *sql.Tx, csvName string) error {
 
 func getTailFromBundle(tx *sql.Tx, head string) (bundles []string, err error) {
 	getReplacesSkips := `SELECT replaces, skips FROM operatorbundle WHERE name=?`
-	isDefaultChannelHead := `SELECT head_operatorbundle_name FROM channel
-							INNER JOIN package ON channel.name = package.default_channel
-							WHERE channel.head_operatorbundle_name = ?`
+	isDefaultChannelHead := `SELECT head_operatorbundle_name FROM package, channel
```

Comment:
I took this issue as an opportunity to dust off my SQL sk1llz and here's what I've found:

The issue in the original query seems to be that the inner join is only made against the channel name and there are two channels (2.x and alpha). So, it's also joining the alpha rows with packages that have a default channel of alpha. An alternative query here would be to join on both the package name and the default channel. E.g.

```
SELECT head_operatorbundle_name FROM channel
INNER JOIN package ON channel.package_name = package.name AND channel.name = package.default_channel
WHERE channel.head_operatorbundle_name = ?
```

I believe the above and what you propose are equivalent in performance and intent. However, [this](https://stackoverflow.com/questions/13476029/multiple-table-select-vs-join-performance) StackOverflow entry seems to suggest that the inner join syntax is recommended. Not that this means you should change anything. Just relating what I've found ^^

Reply:
Oh, I didn't see the previous discussion on this. Feel free to ignore the comments above ^^